---
description: "Papers"
modified: 2021-12-29
tags: [paper]
---

<div class="container">
  <div class="row">
    <!-- sidebar, which will move to the top on a small screen -->
    <div class="col-sm-4">
      <nav id="toc" data-spy="affix" data-toggle="toc"></nav>
    </div>
    <!-- main content area -->
    <div class="col-sm-12">

    <div class="text-center">
      <h3 data-toc-skip> Please cite these papers in your work! </h3>
    </div>
<br>

<hr class="featurette-divider">

<div class="row featurette">
  <!-- <div class="col-md-3">
    <img class="featurette-image img-responsive center-block" src="{{site.baseurl}}/images/paper1.jpg" alt="Paper 1">
  </div> -->
  <div class="col-md-9">
    <h3 data-toc-skip>Resolution of the curse of dimensionality in single-cell RNA sequencing data analysis</h3>
    <p>Yusuke Imoto†, Tomonori Nakamura†, Emerson G. Escolar, Michio Yoshiwaki, Yoji Kojima, Yukihiro Yabuta, Yoshitaka Katou, Takuya Yamamoto, Yasuaki Hiraoka*, Mitinori Saitou*.
    <p><em>bioRxiv </em> May 2, 2022. 
    <p>doi: https://doi.org/10.1101/2022.05.02.490246
    <br>
    <p>Single-cell RNA sequencing (scRNA-seq) can determine gene expression in numerous individual cells simultaneously, promoting progress in the biomedical sciences. However, scRNA-seq data are high-dimensional with substantial technical noise, including dropouts. During analysis of scRNA-seq data, such noise engenders a statistical problem known as the curse of dimensionality (COD). Based on high-dimensional statistics, we herein formulate a noise reduction method, RECODE (resolution of the curse of dimensionality), for high-dimensional data with random sampling noise. We show that RECODE consistently eliminates COD in relevant scRNA-seq data with unique molecular identifiers. RECODE does not involve dimension reduction and recovers expression values for all genes, including lowly expressed genes, realizing precise delineation of cell-fate transitions and identification of rare cells with all gene information. Compared to other representative imputation methods, RECODE employs different principles and exhibits superior overall performance in cell-clustering and single-cell level analysis. The RECODE algorithm is parameter-free, data-driven, deterministic, and high-speed, and notably, its applicability can be predicted based on the variance normalization performance. We propose RECODE as a general strategy for preprocessing noisy high-dimensional data.

    <div class="text-center">
      <a class="btn btn-default btn-sm" href="https://doi.org/10.1101/2022.05.02.490246" role="button">HTML</a>
      <a class="btn btn-default btn-sm" href="https://www.biorxiv.org/content/10.1101/2022.05.02.490246v1.full.pdf" role="button">PDF</a>
    </div>
  </div>
</div>

<div class="row featurette">
  <a class="center">
    <img width=170 class="media-object" src="{{ "/images/NoImage.png"| relative_url }}">
  </a>
</div>

<hr class="featurette-divider">


  </div>
</div>

</div>
</div>
</div>
